src/core/App/moviesBrowserSlice.js

```diff
@@ -0,0 +1,39 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const MoviesBrowserSlice = createSlice({
+    name: "MoviesBrowser",
+    initialState: {
+        status: "initial",
+        genres: null,
+        configuration: null,
+    },
+    reducers: {
+        fetchMoviesBrowserLoading: () => ({
+            status: "loading",
+        }),
+        fetchMoviesBrowserSuccess: (_, { payload: moviesBrowserData}) => ({
+            status: "success",
+            genres: moviesBrowserData.moviesBrowserGenres,
+            // configuration: MoviesBrowserData.configuration,
+        }),
+        fetchMoviesBrowserError: () => ({
+            status: "error",
+            movieData: null,
+            movieCredits: null,
```

Comment:
should be: 
genres: null,
configuration: null,

Reply:
ok